Documentation/security/network/encryption-ipsec.rst

```diff
@@ -328,6 +328,21 @@ errors.
    from the anti-reply oseq on the destination node. This can be fixed by
    properly performing a new key rotation.
 
+  * KVStore Mode (e.g., etcd): XFRM state inconsistencies can occur if the KVStore
```

Comment:
As a matter of structure, all of the other paragraphs in this section seem to be about specific errors returned by Xfrm. However, these two new paragraphs are general statements, and they are located in the middle of the section. Probably it makes more sense to move these paragraphs out to either (a) the end of this section or (b) as part of a new subsection.